sdk/aot/azure-aot-graalvm-samples/pom.xml

```diff
@@ -43,8 +43,14 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <legal>
-            <![CDATA[[INFO] Any downloads listed may be third party software. Microsoft grants you no rights for third party software.]]></legal>
+        <legal><![CDATA[[INFO] Any downloads listed may be third party software. Microsoft grants you no rights for third party software.]]></legal>
+
+        <!-- Sample project, skip all linting. -->
+        <checkstyle.skip>true</checkstyle.skip>
+        <codesnippet.skip>true</codesnippet.skip>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
+        <revapi.skip>true</revapi.skip>
+        <spotbugs.skip>true</spotbugs.skip>
```

Comment:
Any reason to not have linting for this? Maybe we can skip revapi for this one but we can keep the others.

Reply:
Yeah, we can just skip RevApi (which seems to be very costly with GraalVM)